Neos.ContentRepository.Core/Classes/SharedModel/Workspace/WorkspaceName.php

```diff
@@ -23,6 +23,10 @@
  */
 final class WorkspaceName implements \JsonSerializable
 {
+    public const MAX_LENGTH = 30;
+
+    private const PATTERN = '/^[a-z][a-z0-9\-]{0,' . (self::MAX_LENGTH - 1) . '}$/';
```

Comment:
clever. but maybe tooo clever :D

Reply:
why?

Reply:
there is no ide support anymore i assume and its a little obscured but there are tests so +1